.gitignore

```diff
@@ -15,4 +15,8 @@ build/
 [Oo]bj/
 [Rr]elease/
 _[Rr]e[Ss]harper.*/
-*.tss
\ No newline at end of file
+*.tss
```

Comment:
I think you're blind drunk, using some Swedish/English that us English people don't get or you're taking the piss. I'm going for the latter.